surfsense_web/hooks/useConnectorEditPage.ts

```
@@ -1,259 +1,424 @@
-                }
-                break;
-            case 'DISCORD_CONNECTOR':
-                if (formData.DISCORD_BOT_TOKEN !== originalConfig.DISCORD_BOT_TOKEN) {
-                    if (!formData.DISCORD_BOT_TOKEN) { toast.error("Discord Bot Token cannot be empty."); setIsSaving(false); return; }
-                    newConfig = { DISCORD_BOT_TOKEN: formData.DISCORD_BOT_TOKEN };
-                }
-                break;
-        }
+      switch (connector.connector_type) {
+        case "GITHUB_CONNECTOR":
+          const currentPatInForm = patForm.getValues("github_pat");
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)